wallet/initdb.d

```diff
@@ -0,0 +1,75 @@
+create table if not exists wallets
+(
+    address text      not null,
```

Comment:
let's use `id` with UUID type instead of `address` and postpone `id` with bigserial type.